src/general/StageHUDBase.cs

```diff
@@ -1390,4 +1456,12 @@ private void UpdatePausePrompt()
             pauseInfo.Visible = false;
         }
     }
+
+    private void UpdateFossilisationButtons()
+    {
+        foreach (FossilisationButton button in fossilisationButtonLayer.GetChildren())
```

Comment:
I'd prefer to have an early return in this method if the fossilizationButtonLayer is not visible.